book/src/group-snapshot-restore-feature.md

```diff
@@ -4,7 +4,7 @@
 
 Status | Min K8s Version | Max K8s Version | snapshot-controller Version | snapshot-validation-webhook Version | CSI external-snapshotter sidecar Version | external-provisioner Version
 --|--|--|--|--|--|--
-Alpha  | 1.27 | - | 7.0+ | 7.0+ | 7.0+ | 4.0+
+Beta  | 1.27 | - | 8.2+ | 8.2+ | 8.2+ | 4.0+
```

Comment:
We should keep the Alpha entry and adds the max K8s version to 1.32 for Alpha. The Min K8s verision should be 1.32 as that's when it moved to Beta.

Reply:
updated the doc, PTAL